hw_c/5_hw_c_sort_station/station.c

```diff
@@ -0,0 +1,77 @@
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+#include "5_hw_c_stack/stack.h"
+
+void appendStr(char* str, char c)
+{
+    int len = strlen(str);
+    str[len] = c; 
+    str[len + 1] = '\0';
+}
+
+int isOperator(char c) {
+    return c == '(' || c == '+' || c == '-' || c == '*' || c == '/' || c == ')';
+}
+
+// Функция для определения приоритета операторов
+int prioritet(char c)
+{
+    if (c == '+' || c == '-'){
+        return 1;
+    }
+    if (c == '*' || c == '/') {
+        return 2;
+    }
+    return 0;
+}
+
+int main(void)
+{
+    char string[100];
+    char result[100];
+    printf("Введите строку: ");
+    fgets(string, sizeof(string), stdin);
+    OBJ* top = NULL;
+    for (int i = 0; string[i] != '\0'; i++) {
+        if (isdigit(string[i])) {
+            appendStr(result, string[i]);
+            appendStr(result, ' ');
+            continue;
+        }
+        if (isOperator(string[i]) == 1) {
```

Comment:
Здесь явное сравнение с единицей никак не украшает код. А если его опустить, то условие будет читаться естетсвенно, как предложение:
```suggestion
        if (isOperator(string[i])) {
```

А вообще, лучше для таких вещей использовать `bool`.